packages/crayons-styles/src/index.js

```diff
@@ -0,0 +1,50 @@
+import crayons from '../dist/design-tokens/crayons.json';
+const base = '$base.';
+
+function getValue(value) {
```

Comment:
`getValue` and `convertTokenToString` are getting repeated in 2 files. can u reuse them instead?